internal/encoding/block/from_orc.go

```diff
@@ -148,5 +148,26 @@ func convertToString(value interface{}) (string, bool) {
 		return strconv.FormatInt(valueInt, 10), true
 	}
 
+	valueInt32, ok := value.(int32)
```

Comment:
Instead of checking one by one, use a type switch (see https://tour.golang.org/methods/16)